hazenlib/spatial_resolution.py

```diff
@@ -174,9 +174,9 @@ def thresh_image(img, bound=150):
 
 
 def find_square(img):
-    cnts = cv.findContours(img.copy(), cv.RETR_LIST, cv.CHAIN_APPROX_SIMPLE)
+    cnts = cv.findContours(img.copy(), cv.RETR_LIST, cv.CHAIN_APPROX_SIMPLE)[0]
```

Comment:
Presumably this is to take into account the different OpenCV version? But you haven't updated the pip requirements version here (unless that's what environment.yml is doing??)

Reply:
Correct – see comment about on .yml file.